src/app/components/Tag/Tag.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styles from './Tag.module.css';
+
+type TagProps = {
+  count?: string;
+  active?: boolean;
+  children: React.ReactNode;
+};
+
+export default function Tag({
+  count,
+  active,
+  children,
+}: TagProps): JSX.Element {
+  return (
+    <button
```

Comment:
I would definitely add an `onClick` prop, so we can assign a function when using this tag. According to the design, I'd expect some sort of filter function